tests/unit/base/trained_model_serialization_unittest.cc.py

```diff
@@ -16,11 +16,13 @@
     'CMultitaskL12LogisticRegression', 'CMultitaskLeastSquaresRegression',
     'CMultitaskTraceLogisticRegression', 'CMultitaskClusteredLogisticRegression',
     'CLatentSVM', 'CLatentSOSVM', 'CDomainAdaptationSVMLinear',
+    'CLinearLatentMachine',
```

Comment:
Whats up with these changes?

Reply:
It is kinda wrong but I didn't want to research more on that. For some reason it started generating tests for these ones that are abstract.

Reply:
ok well it shouldnt ...